server/chat-plugins/room-events.ts

```diff
@@ -222,6 +229,25 @@ export const commands: ChatCommands = {
 			this.modlog('ROOMEVENT', null, resultString);
 			return this.sendReply(resultString);
 		},
+		enable(target, room, user, cmd) {
+			if (!this.can('gamemanagement')) return false;
```

Comment:
again, why are you not passing a room to this? this currently requires global auth.

Reply:
dense mistake